internal/gitrepo/gitrepo.go

```diff
@@ -257,6 +260,30 @@ func (r *LocalRepository) ChangedFiles() ([]string, error) {
 	return changedFiles, nil
 }
 
+// NewAndDeletedFiles returns a list of files that are new or deleted.
+func (r *LocalRepository) NewAndDeletedFiles() ([]string, error) {
+	slog.Debug("Getting new and deleted files")
```

Comment:
nit: I believe we do this at the Info level for other methods. Same for other new methods.